Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -1,594 +1,354 @@
+  --boclist-horizontal-layout: 80% 20%;
 }
 
-div.bocListNavigator a[href]:active
-{
-  color: Black;
-  border: solid 1px #569CE0;
-  background-color: #C5E0FC;
-}
-
-div.bocListNavigator img
-{
-  padding-left: 0.35em;
-  padding-right: 0.35em;
-  vertical-align: -0.1em;
-}
-
-div.bocListNavigator a[id$="_First"] img,
-div.bocListNavigator a[id$="_Last"] img
-{
-  height: 16px;
-  width: 16px;
-}
-
-div.bocListNavigator a[id$="_Previous"] img,
-div.bocListNavigator a[id$="_Next"] img
-{
-  height: 16px;
-  width: 16px;
-}
-
-div.bocListNavigator a[href]:focus img,
-div.bocListNavigator a[href]:hover img,
-div.bocListNavigator a[href]:active img
-{
-  -webkit-filter: brightness(0);
-  filter: brightness(0);
-  filter: progid:DXImageTransform.Microsoft.BasicImage(grayscale=1);
-}
-
-/*  Style applied to the BocList's menu block. */
-div.bocListMenuBlock
-{
-  display: block;
-  float: right;
-  width: 10em;
-}
-
-div.bocListMenuBlock > div
+/*region root*/
+div.bocList
 {
-  margin-left: 0.5em;
+  display: grid;
+  grid-template-columns: var(--boclist-horizontal-layout);
+  height: auto;
 }
 
-/*  Style applied to the list-area (table and navigation block) of the BocList.  */
-div.bocListTableBlock
+div.bocList a
 {
-  display: block;
-  width: auto;
-  overflow: hidden;
+  text-decoration: none;
 }
 
-/* Style applied to the list-area (table and navigation block) of the BocList if there is a menu block. */
-div.bocListTableBlock.hasMenuBlock
+div.bocList a:hover
 {
-  border-right: 1px solid #AAAAAA;
+  text-decoration: none;
 }
+/*endregion*/
 
-/*  use JS to apply hasWidth and hasHeight depending on bocList-element on runtime */
-/*  Style applied to the list-area (table and navigation block) of the BocList when the BocList has a defined width or height.  */
-div.bocList.hasDimensions div.bocListTableBlock
+/*region bocListTableContainer*/
+div.bocList div.bocListTableBlock.hasNavigator div.bocListTableContainer
 {
-  position: absolute;
-  top: 0;
-  left: 0;
-  bottom: 0;
-  right: 0;
+  grid-row: 1;
 }
 
-/* Style applied to the list-area (table and navigation block) of the BocList if there is a menu block. */
-div.bocList.hasDimensions div.bocListTableBlock.hasMenuBlock
+div.bocList div.bocListTableBlock div.bocListTableContainer
 {
-  right: 10em; /* width of BocList menu block + some space */
-}
-
-/* Style applied to the div surrounding the BocList's table. */
-div.bocListTableContainer
-{
-  display: block;
-  width: auto;
-  height: auto;
-  background-color: #F0F0F0;
-}
-
-div.bocList.hasDimensions div.bocListTableBlock div.bocListTableContainer
-{
-  position: absolute;
-  top: 0;
-  left: 0;
-  bottom: 0;
-  right: 0;
-  overflow: hidden;
-}
-
-div.bocList.hasDimensions div.bocListTableBlock span:focus + div.bocListTableContainer
-{
-  outline: dotted black 1px;
-  outline: auto 5px -webkit-focus-ring-color;
+  grid-row: 1 / span 2;
+  position: relative;
+  background-color: var(--boclist-tablehead-background-color);
```

Comment:
I think this is no longer needed? I can't get this to shine through.

Please set color to red and re-try in chrome and firefox. if I'm correct, we can drop this.

Reply:
It does shine through after some other proposed changes I made

Reply:
*sigh* okay, let's keep it then...